### ISSUE TITLE
Chopin tradebeans `jboss-cli.sh` permission error

### ISSUE THREAD

**Issue by caizixian** [dacapobench/dacapobench#195]
commit: 69a704e

The benchmark finishes with the following. In partcular, DaCapo attempts to execute a script without `x` permission during `Shutting down Wildfly...`.

As a workaround, one can `chmod +x` the script, which presumably should be done during the build process.

```console
--------------------------------------------------------------------------------
IMPORTANT NOTICE:  This is NOT a release build of the DaCapo suite.
Since it is not an official release of the DaCapo suite, care must be taken when
using the suite, and any use of the build must be sure to note that it is not an
offical release, and should note the relevant git hash.

Feedback is greatly appreciated.   The preferred mode of feedback is via github.
Please use our github page to create an issue or a pull request.
    https://github.com/dacapobench/dacapobench.
--------------------------------------------------------------------------------

Using scaled threading model. 16 processors detected, 16 threads used to drive the workload, in a possible range of [1,2048]
Launching the server
===== DaCapo evaluation-git-69a704e tradebeans starting warmup 1 =====
Resetting database and populating with 1098 stocks...
Populating database with 1536 users...
Finished repopulating database
Running 1024 trade sessions directly on server
Completed 1024 trade sessions comprising 18071 trader actions
	Home ....................  3354 (18.6%)
	Portfolio ...............  1527 ( 8.5%)
	Quote ...................  7480 (41.4%)
	Buy .....................  1663 ( 9.2%)
	Sell ....................  1542 ( 8.5%)
	Update ..................   253 ( 1.4%)
	Register ................   102 ( 0.6%)
	Login ...................  1024 ( 5.7%)
	Logout ..................  1126 ( 6.2%)
===== DaCapo evaluation-git-69a704e tradebeans completed warmup 1 in 8840 msec =====
===== DaCapo tail latency: 50% 93 usec, 90% 3101 usec, 99% 13757 usec, 99.9% 27136 usec, 99.99% 37894 usec, max 39103 usec, measured over 18071 events =====
===== DaCapo evaluation-git-69a704e tradebeans starting warmup 2 =====
Resetting database and populating with 1098 stocks...
Populating database with 1536 users...
Finished repopulating database
Running 1024 trade sessions directly on server
Completed 1024 trade sessions comprising 18071 trader actions
	Home ....................  3354 (18.6%)
	Portfolio ...............  1527 ( 8.5%)
	Quote ...................  7480 (41.4%)
	Buy .....................  1663 ( 9.2%)
	Sell ....................  1542 ( 8.5%)
	Update ..................   253 ( 1.4%)
	Register ................   102 ( 0.6%)
	Login ...................  1024 ( 5.7%)
	Logout ..................  1126 ( 6.2%)
===== DaCapo evaluation-git-69a704e tradebeans completed warmup 2 in 8743 msec =====
===== DaCapo tail latency: 50% 82 usec, 90% 2837 usec, 99% 13103 usec, 99.9% 25066 usec, 99.99% 35238 usec, max 35891 usec, measured over 18071 events =====
===== DaCapo evaluation-git-69a704e tradebeans starting warmup 3 =====
Resetting database and populating with 1098 stocks...
Populating database with 1536 users...
Finished repopulating database
Running 1024 trade sessions directly on server
Completed 1024 trade sessions comprising 18071 trader actions
	Home ....................  3354 (18.6%)
	Portfolio ...............  1527 ( 8.5%)
	Quote ...................  7480 (41.4%)
	Buy .....................  1663 ( 9.2%)
	Sell ....................  1542 ( 8.5%)
	Update ..................   253 ( 1.4%)
	Register ................   102 ( 0.6%)
	Login ...................  1024 ( 5.7%)
	Logout ..................  1126 ( 6.2%)
===== DaCapo evaluation-git-69a704e tradebeans completed warmup 3 in 8699 msec =====
===== DaCapo tail latency: 50% 79 usec, 90% 2777 usec, 99% 12279 usec, 99.9% 23675 usec, 99.99% 35884 usec, max 40962 usec, measured over 18071 events =====
===== DaCapo evaluation-git-69a704e tradebeans starting warmup 4 =====
Resetting database and populating with 1098 stocks...
Populating database with 1536 users...
Finished repopulating database
Running 1024 trade sessions directly on server
Completed 1024 trade sessions comprising 18071 trader actions
	Home ....................  3354 (18.6%)
	Portfolio ...............  1527 ( 8.5%)
	Quote ...................  7480 (41.4%)
	Buy .....................  1663 ( 9.2%)
	Sell ....................  1542 ( 8.5%)
	Update ..................   253 ( 1.4%)
	Register ................   102 ( 0.6%)
	Login ...................  1024 ( 5.7%)
	Logout ..................  1126 ( 6.2%)
===== DaCapo evaluation-git-69a704e tradebeans completed warmup 4 in 8678 msec =====
===== DaCapo tail latency: 50% 80 usec, 90% 2781 usec, 99% 12889 usec, 99.9% 23615 usec, 99.99% 36839 usec, max 36991 usec, measured over 18071 events =====
===== DaCapo evaluation-git-69a704e tradebeans starting =====
Resetting database and populating with 1098 stocks...
Populating database with 1536 users...
Finished repopulating database
Running 1024 trade sessions directly on server
Completed 1024 trade sessions comprising 18071 trader actions
	Home ....................  3354 (18.6%)
	Portfolio ...............  1527 ( 8.5%)
	Quote ...................  7480 (41.4%)
	Buy .....................  1663 ( 9.2%)
	Sell ....................  1542 ( 8.5%)
	Update ..................   253 ( 1.4%)
	Register ................   102 ( 0.6%)
	Login ...................  1024 ( 5.7%)
	Logout ..................  1126 ( 6.2%)
===== DaCapo evaluation-git-69a704e tradebeans PASSED in 8995 msec =====
===== DaCapo tail latency: 50% 79 usec, 90% 2875 usec, 99% 12335 usec, 99.9% 24468 usec, 99.99% 37341 usec, max 42188 usec, measured over 18071 events =====
Exception initializing server: java.io.IOException: Cannot run program "/usr/share/benchmarks/dacapo/dacapo-evaluation-git-69a704e/dat/lib/daytrader/wildfly-20.0.1.Final/bin/jboss-cli.sh": error=13, Permission deniedjava.io.IOException: Cannot run program "/usr/share/benchmarks/dacapo/dacapo-evaluation-git-69a704e/dat/lib/daytrader/wildfly-20.0.1.Final/bin/jboss-cli.sh": error=13, Permission denied
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at org.dacapo.daytrader.DaCapoServerRunner.shutdown(DaCapoServerRunner.java:84)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.dacapo.daytrader.Launcher.shutdown(Launcher.java:135)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.dacapo.harness.Tradebeans.cleanup(Tradebeans.java:69)
	at org.dacapo.harness.TestHarness.runBenchmark(TestHarness.java:227)
	at org.dacapo.harness.TestHarness.main(TestHarness.java:170)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at Harness.main(Unknown Source)
Caused by: java.io.IOException: error=13, Permission denied
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 19 more
```

**Comment by caizixian** [comment]
Still an issue for the latest commit. Reproduce with the following command.
`/usr/lib/jvm/temurin-11-jdk-amd64/bin/java -server -Xms1G -Xmx1G --add-exports java.base/jdk.internal.ref=ALL-UNNAMED -cp /usr/share/benchmarks/dacapo/dacapo-evaluation-git-6e411f33.jar Harness -n 5 -s default tradesoap`

**Comment by caizixian** [comment]
Simpler repro attached.

```console
/usr/lib/jvm/temurin-11-jdk-amd64/bin/java -jar /usr/share/benchmarks/dacapo/dacapo-evaluation-git-6e411f33.jar tradesoap
--------------------------------------------------------------------------------
IMPORTANT NOTICE:  This is NOT a release build of the DaCapo suite.
Since it is not an official release of the DaCapo suite, care must be taken when
using the suite, and any use of the build must be sure to note that it is not an
offical release, and should note the relevant git hash.

Feedback is greatly appreciated.   The preferred mode of feedback is via github.
Please use our github page to create an issue or a pull request.
    https://github.com/dacapobench/dacapobench.
--------------------------------------------------------------------------------

Using scaled threading model. 24 processors detected, 24 threads used to drive the workload, in a possible range of [1,128]
Version: daytrader svn-1904225 wildfly 26.1.2
Launching the server
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.wildfly.extension.elytron.SSLDefinitions (jar:file:/usr/share/benchmarks/dacapo/dacapo-evaluation-git-6e411f33/dat/lib/daytrader/wildfly-26.1.2.Final/modules/system/layers/base/org/wildfly/extension/elytron/main/wildfly-elytron-integration-18.1.2.Final.jar!/) to method com.sun.net.ssl.internal.ssl.Provider.isFIPS()
WARNING: Please consider reporting this to the maintainers of org.wildfly.extension.elytron.SSLDefinitions
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
===== DaCapo evaluation-git-6e411f33 tradesoap starting =====
Resetting database and populating with 1098 stocks...
Populating database with 768 users...
Finished repopulating database
Running 512 trade sessions from client via soap
Starting 8840 requests...
Completed requests
Completed 512 trade sessions comprising 8840 trader actions
	Home ....................  1633 (18.5%)
	Portfolio ...............   737 ( 8.3%)
	Quote ...................  3633 (41.1%)
	Buy .....................   832 ( 9.4%)
	Sell ....................   750 ( 8.5%)
	Update ..................   123 ( 1.4%)
	Register ................    54 ( 0.6%)
	Login ...................   512 ( 5.8%)
	Logout ..................   566 ( 6.4%)
===== DaCapo evaluation-git-6e411f33 tradesoap PASSED in 9956 msec =====
===== DaCapo simple tail latency: 50% 2031 usec, 90% 11521 usec, 99% 30015 usec, 99.9% 52439 usec, 99.99% 68266 usec, max 68266 usec, measured over 8840 events =====
===== DaCapo metered tail latency: 50% 79462 usec, 90% 126758 usec, 99% 140677 usec, 99.9% 160660 usec, 99.99% 174759 usec, max 174759 usec, measured over 8840 events =====
Exception initializing server: java.io.IOException: Cannot run program "/usr/share/benchmarks/dacapo/dacapo-evaluation-git-6e411f33/dat/lib/daytrader/wildfly-26.1.2.Final/bin/jboss-cli.sh": error=13, Permission deniedjava.io.IOException: Cannot run program "/usr/share/benchmarks/dacapo/dacapo-evaluation-git-6e411f33/dat/lib/daytrader/wildfly-26.1.2.Final/bin/jboss-cli.sh": error=13, Permission denied
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at org.dacapo.daytrader.DaCapoServerRunner.shutdown(DaCapoServerRunner.java:84)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.dacapo.daytrader.Launcher.shutdown(Launcher.java:133)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.dacapo.harness.Tradesoap.cleanup(Tradesoap.java:64)
	at org.dacapo.harness.TestHarness.runBenchmark(TestHarness.java:227)
	at org.dacapo.harness.TestHarness.main(TestHarness.java:170)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at Harness.main(Unknown Source)
Caused by: java.io.IOException: error=13, Permission denied
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 19 more
```

**Comment by steveblackburn** [comment]
Fixed in 05f56c9
